client/lib/post-types-list/list.js

```diff
@@ -73,7 +75,7 @@ PostTypesList.prototype.fetch = function( siteId ) {
  * @return {array} services
  **/
 PostTypesList.prototype.parse = function( data ) {
-	return data.post_types;
+	return values( omit( data, '_headers' ) );
```

Comment:
This is not coming from the API response directly.  With the HTTP envelope enabled, our v1 API returns `body, code, headers` properties, and WP-API returns `body, headers, status` properties.

I haven't looked to see which layer is adding the `_headers` property, but we do [similar removals](https://github.com/Automattic/wp-calypso/search?utf8=%E2%9C%93&q=_headers) in other parts of the code.  In this case we didn't need to do it before because there was a `post_types` property on the body rather than the body being the object itself.